source/adios2/operator/compress/CompressZFP.cpp

```diff
@@ -63,6 +63,7 @@ size_t CompressZFP::Compress(const void *dataIn, const Dims &dimensions,
 
     zfp_field_free(field);
     zfp_stream_close(stream);
+    free(bitstream);
```

Comment:
This is my bad. I might have misunderstood what the (`zfp_stream_set_bit_stream` function)[https://github.com/ornladios/ADIOS2/blob/master/source/adios2/operator/compress/CompressZFP.cpp#L50] does. In any case, I'd use (`zfp_stream_close`)[https://zfp.readthedocs.io/en/release0.5.5/bit-stream.html#c.stream_close] instead of free for consistency.

Reply:
Good point. I tried this but got:

```
/home/4nt/ADIOS2/source/adios2/operator/compress/CompressZFP.cpp:66:5: error: no matching function for call to 'zfp_stream_close'
    zfp_stream_close(bitstream);
    ^~~~~~~~~~~~~~~~
/usr/local/include/zfp.h:153:1: note: candidate function not viable: cannot convert argument of incomplete type 'bitstream *' to 'zfp_stream *' for 1st argument
zfp_stream_close(
^
1 error generated.
```

I therefore looked at the [zfp example](https://github.com/LLNL/zfp/blob/develop/examples/simple.c) and found that the correct call is `stream_close`.